{{ .ProjectSnake }}/BUILD.bazel

```diff
@@ -5,6 +5,9 @@ load("@npm//:defs.bzl", "npm_link_all_packages")
 {{ end }}
 {{- if .Computed.python }}
 load("@rules_python//python:pip.bzl", "compile_pip_requirements")
+load("@pip//:requirements.bzl", "all_whl_requirements")
+load("@rules_python_gazelle_plugin//manifest:defs.bzl", "gazelle_python_manifest")
+load("@rules_python_gazelle_plugin//modules_mapping:def.bzl", "modules_mapping")
```

Comment:
These loads should be lexicographically sorted, otherwise the generated repo will not be lint clean.

Reply:
I think you're actually telling me "why is the CI green on `main` when you've broken the rules" and I'll tell you "*grumble* why should rules apply to me?" and then I'll send you a PR to fix it.

Reply:
Well the ones you currently have are sorted correctly, so I'd expect main to be green here, but the new ones are not sorted correctly.

Reply:
The new order should be 
```
load("@aspect_rules_js//js:defs.bzl", "js_library")
load("@npm//:defs.bzl", "npm_link_all_packages")
load("@pip//:requirements.bzl", "all_whl_requirements")
load("@rules_python_gazelle_plugin//manifest:defs.bzl", "gazelle_python_manifest")
load("@rules_python_gazelle_plugin//modules_mapping:def.bzl", "modules_mapping")
load("@rules_python//python:pip.bzl", "compile_pip_requirements")
```

So you'll have to interleave or postprocess etc.

Reply:
The whitespace makes it hard though. You have to be exceptionally careful between `{{ expr }}` and `{{- expr}}` or you'd break it

Reply:
Okay just fixing the easy way for this PR.

Reply:
See #47 for the assertion :)